install/installer/pkg/components/spicedb/data/schema.yaml

```diff
@@ -32,3 +32,17 @@ schema: |-
   }
 
   definition service {}
+
+# relationships to be used for assertions
+relationships: >-
+  organisation:org1#owner@user:user1
+
+assertions:
```

Comment:
For now, this is pretty limited but this PR focuses on setting it up. We'll iterate on the model and assertions in a follow-up.